pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/DataLoader.java

```diff
@@ -666,6 +668,7 @@ public void loadReleaseSurveyData(Handle handle, ParticipantData participantData
             for (Institution institution : institutionList) {
 
                 String guid = getMedicalProviderGuid(handle);
+                long countryAdressId = getCountryAdressId(handle, institution.getCountry());
```

Comment:
The `attach()` call made by `getCountryAddressId()` is an expensive call- try to avoid calling it multiple times withing the scope of a single method.